src/org/opensolaris/opengrok/history/GitHistoryParser.java

```diff
@@ -142,36 +140,45 @@ private void process(BufferedReader in) throws IOException {
         if (entry != null) {
             entries.add(entry);
         }
-
-        history.setHistoryEntries(entries);
     }
 
     /**
      * Parse the history for the specified file.
      *
      * @param file the file to parse history for
-     * @param repos Pointer to the SubversionReporitory
+     * @param repos Pointer to the GitRepository
      * @param sinceRevision the oldest changeset to return from the executor, or
      *                      {@code null} if all changesets should be returned
      * @return object representing the file's history
      */
     History parse(File file, Repository repos, String sinceRevision) throws HistoryException {
         myDir = repos.getDirectoryName() + File.separator;
         repository = (GitRepository) repos;
+        RenamedFilesParser parser = new RenamedFilesParser();
         try {
             Executor executor = repository.getHistoryLogExecutor(file, sinceRevision);
             int status = executor.exec(true, this);
 
             if (status != 0) {
-                throw new HistoryException("Failed to get history for: \"" +
-                        file.getAbsolutePath() + "\" Exit code: " + status);
+                throw new HistoryException("Failed to get history for: \""
+                        + file.getAbsolutePath() + "\" Exit code: " + status);
```

Comment:
This is what I get when I autoformat in netbeans with default settings. I changed those lines anyway to use `String.format` and not string composition.